clang/test/Modules/pr130712.cppm

```diff
@@ -0,0 +1,31 @@
+// RUN: split-file %s %t
+
+// There are two requirements here to result in the owner of a macro being null.
```

Comment:
Which macro has null owner module in this test?

Reply:
A_H